imapautofiler/client.py

```diff
@@ -113,15 +114,23 @@ def expunge(self):
     def close(self):
         "Close the connection, flushing any pending changes."
 
-
+'/Users/olsenmo/Desktop/internal.pem'
```

Comment:
Was this literal string for debugging?

Reply:
Looks like something got pasted in the wrong window. Sorry about that, I'll remove.